release_notes.md

```diff
@@ -1,3 +1,58 @@
+
+Another new feature is the openmdao.api.slicer which helps specify src_indices for connections or indices for design variables, objectives, and constraints.
+This is intended to allow the user to easily connect portions of an output to an input (such as selecting an individual column from a matrix, for instance).
+The old method of providing an explicit list of indices should still work. However, the new slicer gives greater functionality since it allows for the use of general slicing syntax such as `om.slicer[0:10:1, :]`.
+Using the slicer object is now considered the new best coding practice for giving src_indices or indices arguments.
+
+We hope these changes will reduce the development burden on our users.
+As this is a significant change, some issues may have slipped through our testing, and any feedback is always welcomed via issues on the GitHub repository.
+
+## Backwards Incompatible API Changes:
+
+- Fix a typo in user_terminate_signal option, deprecated the old. #1469
+- Remove support for factorial function in ExecComp. #1483
+
+## Backwards Incompatible NON-API Changes:
+
+- _all_subsystem_iter has been removed.  Users should use the systems_iter method instead.
+
+## New Features:
+
+- N2 viewer changes to improve performance. #1475
+- Local table of contents (navigation) added to docs sidebar. #1477
+- N2 now uses coloring info by default to show dependence in each component. #1478
+- Added test to capture error message when running in parallel with non distributed components. #1484
+- N2 adds a spinner to give the user an indication when it isn't finished rendering. #1485
+- All case recorder files now always contain system options information about all systems in the model. #1486
+- Model data now compressed when saved in N2 HTML file. #1490
+- Added support for om.slicer as a way to pass in slices to src_indices/indices arguments in add_design_var, add_response, add_constraint, promotes, and connect. #1491
+- Added flag for `under_approx` to let the user know when a system is operating under any derivative approximation scheme, and a new counter `iter_count_without_approx`. #1492
+- Automatically add a single, user-hidden, top-level IndepVarComp to provide outputs for every unconnected input in the model.
+
+## Bug Fixes:
+
+- Fix bug where exception was raised with while printing bounds violations via "print_bound_enforce" option on linesearch if one side of an output was unbounded. #1466
+- Fixed an exception when using AddSubtractComp constructor instead of its add_equation method. #1474
+- Minor fixes for Appveyor CI. #1495
+- Problem.record() provides a useful error message if called before final_setup. #1545
+- Fixed a bug that was obliterating options passed to BalanceComp, and added tests for better coverage. #1546
+- Added quotes to install command in docs to prevent problems in some shell environments. #1547
+
 **********************************
```

Comment:
Missing: New Features
-------------------------
Allow the linear algebra components (DotProductComp, CrossProductComp, MatrixVectorProductComp, VectorMagnitudeComp) to each perform multiple calculations. #1503